graviti/utility/user.py

```diff
@@ -378,6 +378,8 @@ def __init__(
 
         iterable = items.items() if isinstance(items, Mapping) else items
         for key, value in iterable:
+            self._check_key(key)
+            self._check_value(value)
```

Comment:
Do we need to check the value type in base class?

Reply:
In the base class, it does not actually check the value type, because the `_check_value` method is passed. It is for the choice of the subclass to check the value or not.